geniza/corpus/views.py

```diff
@@ -661,18 +661,34 @@ def page_title(self):
     def get_context_data(self, **kwargs):
         """Pass annotation configuration and TinyMCE API key to page context"""
         context_data = super().get_context_data(**kwargs)
+
+        # get source uri if source_pk present in kwargs (i.e. editing an existing transcription),
+        # and the source exists
+        source_uri = ""
+        source_pk = self.kwargs.get("source_pk", None)
+        try:
+            source = Source.objects.get(pk=source_pk)
+            source_uri = source.uri
+        except Source.DoesNotExist:
+            # NOTE: Should this raise a 404 if source_pk is not None?
+            # TODO: If source_pk is None, instantiate the scholarship record choice form
+            pass
```

Comment:
Shouldn't try to request a source when `source_pk` is `None`; add a comment to clarify that's a new transcription